### PR TITLE
Update Remove-NetIPAddress.md

### PR DESCRIPTION
**Pull request by JulianTurner**
changed the description of the comment because it was missleading and incomplete

### CODE REVIEW

**Comment by e0i** [comment]
@JulianTurner Please change the target branch of the PR from `live` to `master` to continue with the review. Thank you.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/nettcpip/Remove-NetIPAddress.md [hunk 1]
```diff
@@ -61,17 +61,19 @@ This command adds a new IP address.
 
 ### Example 2: Remove an IP address using a pipeline
 ```
+This command removes all of the IP addresses with the address 192.168.0.1.
 PS C:\>Get-NetIPAddress -IPAddress 192.168.0.1 | Remove-NetIPAddress
 ```
```
Comment (dariomws) [inline docset/windows/nettcpip/Remove-NetIPAddress.md:66]:
```
PS C:\>Get-NetIPAddress -IPAddress 192.168.0.1 | Remove-NetIPAddress
```

This command removes all of the IP addresses with the address 192.168.0.1.

File: docset/windows/nettcpip/Remove-NetIPAddress.md [hunk 2]
```diff
@@ -61,17 +61,19 @@ This command adds a new IP address.
...
 
-This command removes all of the IP addresses with the address 192.168.0.1.
+
 
 ### Example 3: Remove IP addresses by prefix origin
 ```
+This command removes all of the IP addresses that have a manually-configured prefix origin.
 PS C:\>Remove-NetIPAddress -PrefixOrigin Manual
 ```
```
Comment (dariomws) [inline docset/windows/nettcpip/Remove-NetIPAddress.md:74]:
```
PS C:\>Remove-NetIPAddress -PrefixOrigin Manual
```

This command removes all of the IP addresses that have a manually-configured prefix origin.

**Comment by dariomws** [comment]
Example 1 should be updated too.

File: docset/windows/nettcpip/Remove-NetIPAddress.md (hunk 1 above)
Comment (dariomws) [inline docset/windows/nettcpip/Remove-NetIPAddress.md:66]:
```
PS C:\>Get-NetIPAddress -IPAddress 192.168.0.1 | Remove-NetIPAddress
```

This command removes all of the IP addresses with the address 192.168.0.1.

File: docset/windows/nettcpip/Remove-NetIPAddress.md (hunk 2 above)
Comment (dariomws) [inline docset/windows/nettcpip/Remove-NetIPAddress.md:74]:
```
PS C:\>Remove-NetIPAddress -PrefixOrigin Manual
```

This command removes all of the IP addresses that have a manually-configured prefix origin.

**Comment by tseward** [comment]
Agree with @dariomws and when changes are in place, approved.

**Comment by e0i** [comment]
@JulianTurner Would you be able to follow-up with the changes suggested by the reviewers? Thank you.l

**Comment by e0i** [comment]
Closing via replacement PR #1945 targeting the correct branch. Thank you for your contribution.

**Pull request closed without merging**